src/expr/src/vector_op/agg/aggregator.rs

```diff
@@ -215,16 +215,17 @@ pub fn create_agg_state_unary(
         (Count, count, float32, int64, Some(0)),
         (Count, count, float64, int64, Some(0)),
         (Count, count, decimal, int64, Some(0)),
-        (Count, count_str, varchar, int64, Some(0)),
         (Count, count, boolean, int64, Some(0)),
         (Count, count, interval, int64, Some(0)),
         (Count, count, date, int64, Some(0)),
         (Count, count, timestamp, int64, Some(0)),
         (Count, count, time, int64, Some(0)),
+        (Count, count_str, varchar, int64, Some(0)),
         (Count, count_struct, struct_type, int64, Some(0)),
         (Count, count_list, list, int64, Some(0)),
         (Sum, sum, int16, int64, None),
         (Sum, sum, int32, int64, None),
+        (Sum, sum, int64, int64, None),
         (Sum, sum, int64, decimal, None),
```

Comment:
```suggestion
        (Sum, sum, int64, int64, None),
        (Sum, sum, int64, decimal, None),
```

Are the two signatures duplicated?

Reply:
Ok, looks like it was moved from a previous line. Just curious why there are two SUM for i64.

Reply:
Seems mysterious...I think it's never called.

Reply:
If deleting it will not cause any testing failure, then just remove it :)

Reply:
Oh, we can't remove it. Found that `int64 + int64 => decimal` https://github.com/singularity-data/risingwave/blob/62cc3ebb17c38b3800ef666f676d00b9fef22b60/src/frontend/src/expr/agg_call.rs#L125

Reply:
Yes. That's what I mean.

Reply:
The test broke after remove that line. It's used as global phase of `count`.